src/components/BackgroundContainer.tsx

```diff
@@ -1,23 +1,35 @@
-import React from 'react'
-
-type BackgroundContainerProps = { backgroundURL?: string; children?: void }
+import React, { useEffect, useState } from 'react'
+import useCustomWallpaper from 'hooks/useCustomWallpaper'
+import { useClient } from 'cozy-client'
 
 type BackgroundContainerComputedProps = {
   className: string
   style?: { backgroundImage: string }
 }
 
 const makeProps = (
-  backgroundURL?: string
+  backgroundURL: string | null
 ): BackgroundContainerComputedProps => ({
   className: 'background-container',
   ...(backgroundURL && {
     style: { backgroundImage: `url(${backgroundURL})` }
   })
 })
 
-export const BackgroundContainer = ({
-  backgroundURL
-}: BackgroundContainerProps): JSX.Element => (
-  <div {...makeProps(backgroundURL)} />
-)
+export const BackgroundContainer = (): JSX.Element => {
+  const client = useClient()
+  const {
+    fetchStatus,
+    data: { wallpaperLink }
+  } = useCustomWallpaper()
+
+  const [backgroundURL, setBackgroundURL] = useState<string | null>(null)
+
+  useEffect(() => {
+    const { cozyDefaultWallpaper } = client.getInstanceOptions() as {
+      cozyDefaultWallpaper: string
+    }
```

Comment:
I'm wondering if we should define a comment keyword to mark every temporary types that we should remove the day we will fully type cozy-client